kura/org.eclipse.kura.protocol.can.test/src/main/java/org/eclipse/kura/protocol/can/test/CanSocketTest.java

```diff
@@ -152,13 +136,11 @@ public boolean doCanTest() {
             } catch (InterruptedException e) {
             }
         } else {
-            s_logger.info("Wait for a request");
+            logger.info("Wait for a request");
             try {
-
-                cm = this.m_canConnection.receiveCanMessage(-1, 0x7FF);
-
+                cm = this.canConnection.receiveCanMessage(-1, 0x7FF);
```

Comment:
You are right. It would not run successfully to inject the dependencies as in the OSGi container that maven initializes during the test-run, it has to be properly synchronized for dependencies using latches and static modifiers for dependencies.